client/my-sites/controller.js

```diff
@@ -113,6 +113,26 @@ function renderNoVisibleSites( context ) {
 	);
 }
 
+function renderSelectedSiteIsDomainOnly( reactContext, selectedSite ) {
+	const EmptyContentComponent = require( 'components/empty-content' );
+	const { store: reduxStore } = reactContext;
+
+	removeSidebar( reactContext );
+
+	renderWithReduxStore(
+		React.createElement( EmptyContentComponent, {
+			title: i18n.translate( 'This feature is not available for domains' ),
+			line: i18n.translate( 'To use this feature you need to create a site' ),
+			action: i18n.translate( 'Create New Site' ),
+			actionURL: '//dashboard.wordpress.com/wp-admin/index.php?page=my-blogs',
```

Comment:
We need to edit this once the flow exists in M2 anyway, so I think this is fine for now.